libs/common/src/helpers/is-feature-flag-enabled.ts

```diff
@@ -24,3 +25,4 @@ const isFeatureFlagEnabled = (featureFlag: FeatureFlag) => (): boolean => {
 export const isTfmPaymentReconciliationFeatureFlagEnabled = isFeatureFlagEnabled('FF_TFM_PAYMENT_RECONCILIATION_ENABLED');
 
 export const isTfmFacilityEndDateFeatureFlagEnabled = isFeatureFlagEnabled('FF_TFM_FACILITY_END_DATE_ENABLED');
+export const isTfmDealCancellationFeatureFlagEnabled = isFeatureFlagEnabled('FF_TFM_DEAL_CANCELLATION');
```

Comment:
Can we update the unit test? Bit concerning if it's passing when there's been changes